libbeat/docs/newdashboards.asciidoc

```diff
@@ -145,3 +285,33 @@ The output directory where the dashboards and all dependencies will be saved. Th
 NOTE: To make it even easier for you to import and export the dashboards, we've also added support for running the
 following commands in the Beats repository: `make import-dashboards` and `make export-dashboards`.
 
+The output directory has the following structure:
+
+[source,shell]
+--------------
+output/
+    index-pattern/
+    dashboard/
+    visualization/
+    search/
+--------------
+
+[[archive-dashboards]]
+=== Archive your Own Kibana dashboards
+
+The Kibana dashboards for the Elastic Beats are saved under `etc/kibana` directory. To create a zip archive with the
+dashboards, including visualizations and searches and the index pattern, you can run the following command in the Beat
+repository:
+
+[source,shell]
+--------------
+$ make package-setup
+$ make package-dashboards
+--------------
+The used Makefile is part of libbeat, so the previous commands can be used also by the community Beats when the
```

Comment:
Suggest changing this to say: "The Makefile is part of libbeat, which means that community Beats contributors can use the commands shown here to archive dashboards. The dashboards must be available under the `etc/kibana` directory."